packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/ToggleVariants.jsx

```diff
@@ -1,25 +1,26 @@
 import React, { useState } from 'react';
 import StatusView from 'terra-status-view';
+import SingleSelectField from 'terra-form-select/lib/SingleSelectField';
 
 const ToggleVariants = () => {
   const [variant, setVariant] = useState('error');
 
-  const handleOnSelect = (event) => {
-    setVariant(event.target.value);
+  const handleOnSelect = (value) => {
+    setVariant(value);
   };
 
   return (
     <div>
       <StatusView variant={variant} />
-      <fieldset>
-        <legend>Change Variant</legend>
-        <select aria-label="changeVariant" id="statusViewVariant" name="statusViewVariant" value={variant} onChange={handleOnSelect}>
-          <option value="error">Error</option>
-          <option value="no-data">No Data</option>
-          <option value="no-matching-results">No Matching Results</option>
-          <option value="not-authorized">Not Authorized</option>
-        </select>
-      </fieldset>
+      <SingleSelectField label="Change Variant" onChange={handleOnSelect} value={variant} maxWidth="300px" selectId="change-variant-field" placeholder="Select a Variant">
+        <SingleSelectField.Option value="error" display="Error" />
+        <SingleSelectField.Option value="no-data" display="No Results" />
+        <SingleSelectField.Option value="no-matching-results" display="No Matching Result" />
+        <SingleSelectField.Option value="not-authorized" display="Not Authorized" />
+      </SingleSelectField>
+      <p>
+        **  Select a status view variant from the dropdown above.
```

Comment:
nit: can we remove the asterisks here?
```suggestion
        Select a status view variant from the dropdown above.
```

Reply:
yes we can. Thanks